dotnet/src/Extensions/Planning.ActionPlanner/ActionPlanner.cs

```diff
@@ -234,6 +238,10 @@ No parameters.
     }
 
     #region private ================================================================================
+    /// <summary>
```

Comment:
Small formatting improvement:
```suggestion

    /// <summary>
```